helix-view/src/view.rs

```diff
@@ -213,9 +207,10 @@ impl View {
         doc: &Document,
         scrolloff: usize,
     ) -> Option<ViewPosition> {
+        let view_offset = doc.view_data(self.id)?.view_position;
```

Comment:
For this case we can introduce a `pub(crate) fn get_view_data(&self, view_id: ViewId) -> Option<&ViewData>` on Document that doesn't `expect`

Reply:
Done